Makefile.ibi

```diff
@@ -11,7 +12,23 @@ IBI_CLUSTER_NAME ?= $(IBI_VM_NAME)
 IBI_HOSTNAME ?= $(IBI_VM_NAME)
 IBI_INSTALLATION_DISK ?= /dev/vda
 IBI_WORK_DIR ?= ibi-iso-work-dir
-IBI_KUBECONFIG=kubeconfig.ibi
+IBI_INSTALL_CONFIG ?= $(IBI_WORK_DIR)/install-config.yaml
+IBI_KUBECONFIG = $(IBI_WORK_DIR)/auth/kubeconfig
+
+OPENSHIFT_INSTALLER_REPO ?= https://github.com/mresvanis/installer
```

Comment:
Add a comment/TODO explaining why we're using this fork

Reply:
Agree, comment added